Dockerfile

```diff
@@ -12,6 +12,10 @@ COPY pkg pkg
 COPY build.go build.go
 COPY package.json package.json
 
+RUN go get github.com/crewjam/saml
```

Comment:
this is not missing, it's in the vendor dir  https://github.com/grafana/grafana/tree/master/vendor/github.com/crewjam/saml 

and so is https://github.com/grafana/grafana/tree/master/vendor/github.com/stretchr/testify